sim/common/sod/item_effects/phase_5.go

```diff
@@ -61,6 +63,52 @@ func init() {
 		})
 	})
 
+	// https://www.wowhead.com/classic/item=229806/truthbearer
+	// Chance on hit: Increases damage done by 15 and attack speed by 30% for 8 sec.
+	// TODO: Proc rate assumed and needs testing
+	core.NewItemEffect(Truthbearer2H, func(agent core.Agent) {
+		character := agent.GetCharacter()
+
+		procMask := character.GetProcMaskForItem(Truthbearer2H)
+		crusadersZealAura := crusadersZealAura465414(character)
+
+		core.MakeProcTriggerAura(&character.Unit, core.ProcTrigger{
+			Name:              "Truthbearer (Crusaders Zeal)",
+			Callback:          core.CallbackOnSpellHitDealt,
+			Outcome:           core.OutcomeLanded,
+			ProcMask:          procMask,
+			SpellFlagsExclude: core.SpellFlagSuppressWeaponProcs,
+			PPM:               1, // Estimated based on data from WoW Armaments Discord
```

Comment:
I assume this might be a copy paste but could you remove the comments on these two about the armaments discord since they need to be confirmed?